tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -29,8 +29,8 @@ public class UsernamePasswordIntegrationTests
 
         // Http Telemetry Constants
         private static Guid CorrelationId = new Guid("ad8c894a-557f-48c0-b045-c129590c344e");
-        private const string XClientCurrentTelemetryROPC = "2|1003,0|";
-        private const string XClientCurrentTelemetryROPCFailure = "2|1003,0|";
+        private const string XClientCurrentTelemetryROPC = "2|1003,0|0";
```

Comment:
won't adding the new value first break pre-existing queries?

Reply:
Good point, haven't caught that. New values should go at the end.

Reply:
There are no existing queries for regional data currently. But good point to add new fields to end. I will create a PR to fix it. Thanks